workflow/rules/read_mapping.smk

```diff
@@ -60,6 +60,7 @@ rule map_reads:
     threads: 8
     wrapper:
         "0.69.0/bio/bwa/mem"
+        # "v1.1.0/bio/bwa/mem"
```

Comment:
is this still needed?

Reply:
the mapping of results/{date}/nonhuman-reads/se/{sample}.fastq.gz by bwa has never worked for some reason. bwa also does not work on the server i work on, even if i create a conda env manually. However, bwa-mem2 does work and is also supposed to be faster and memory conserving. So I suggest considering to use the according bwa-mem2 wrapper. However, it didn't work out of the box as well when I just exchange the wrappers, so other things need to be adjusted as well I guess.